syncer/online-ddl-tools/online_ddl.go

```diff
@@ -38,6 +39,11 @@ var OnlineDDLSchemes = map[string]func(*tcontext.Context, *config.SubTaskConfig)
 	config.GHOST: NewGhost,
 }
 
+// refactor to reduce duplicate later.
+var (
+	maxCheckPointTimeout = "1m"
```

Comment:
seems to have a duplicate variable in `syncer/checkpoint.go`?

Reply:
Yes, so reduce duplicate later